src/ObjectUtils/include/ObjectUtils/SymbolsFile.h

```diff
@@ -15,11 +15,13 @@
 namespace orbit_object_utils {
 
 struct ObjectFileInfo {
-  // This is the load bias for ELF, for COFF we use ImageBase here, so that our
-  // address computations are consistent between what we do we ELF and for COFF.
+  // For ELF, this is the load bias of the executable segment. For PE/COFF, we use ImageBase here,
+  // so that our address computations are consistent between what we do for ELF and for COFF.
   uint64_t load_bias = 0;
-  // File offset to the beginning of the executable segment. For COFF, this is
-  // the file offset to the beginning of the .text section.
```

Comment:
The content of this comment has changed. It was previously wrong because we use the offset in memory everywhere, including in `CoffFileImpl::GetExecutableSegmentOffset`.